_posts/2021-05-13-paper-cardio.md

```diff
@@ -0,0 +1,23 @@
+---
+site: freiburg
+title: 'New Paper "Novel insights into the electrophysiology of murine cardiac macrophages: relevance of voltage-gated potassium channels"'
+doi: "10.1093/cvr/cvab126"
+tags:
+- paper
+---
+
+
+_Abstract_ - Macrophages (MΦ), known for immunological roles such as phagocytosis and antigen presentation, have been found to electrotonically couple to cardiomyocytes (CM) of the atrio-ventricular node via Cx43, affecting cardiac conduction in isolated mouse hearts. Here, we characterise passive and active electrophysiological properties of murine cardiac resident MΦ, and model their potential electrophysiological relevance for CM.
```

Comment:
we usually name the author and give the group shout out

Reply:
done